tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+project(SOF_FUZZER C)
+
+include(ExternalProject)
+
+set(source_directory "${PROJECT_SOURCE_DIR}/../testbench")
+
+ExternalProject_Add(sof_tb
+	SOURCE_DIR "${source_directory}"
+	PREFIX "${PROJECT_BINARY_DIR}/sof_tb"
+	BINARY_DIR "${PROJECT_BINARY_DIR}/sof_tb/build"
```

Comment:
`"${PROJECT_BINARY_DIR}/sof_tb/build"` is repeated few times, maybe it should be var like `testbench_build_directory`